apps/web/src/utils/config.ts

```diff
@@ -27,7 +27,7 @@ function get<T>(key: string, def?: T): T {
   const value = window.localStorage.getItem(key);
   if (!value && def !== undefined) return def;
 
-  return tryParse(value);
+  return tryParse(value!);
```

Comment:
Can you check the `value` here instead and return `def` if it is undefined?

Reply:
OK